### ISSUE TITLE
Ref not working...

### ISSUE THREAD

**Issue by andrisole92** [enzomanuelmangano/react-native-skia-gesture#5]
No Skia View currently available...

```
	const ref = useCanvasRef();

...
	<Touchable.Canvas
		style={{ width: '100%', height: '50%', backgroundColor: 'black' }}
		ref={ref}>

```

```
    const image = ref.current?.makeImageSnapshot();
```

Obviously works without HOC.

**Comment by enzomanuelmangano** [comment]
Hi @andrisole92 , thanks for opening the issue. I have already a solution for that! I'll update the package with the fix as soon as I can :)

**Comment by enzomanuelmangano** [comment]
Hi @andrisole92 the issue has been fixed on v0.2.0. I'll close the issue then, but feel free to reopen it if you encounter it again 🙌

**Comment by andrisole92** [comment]
Thank you Sir! 🤗
